theano/gpuarray/opt.py

```diff
@@ -1058,31 +1059,36 @@ def local_gpua_advanced_subtensor(op, context_name, inputs, outputs):
 
 @register_opt('fast_compile')
 @op_lifter([tensor.AdvancedIncSubtensor1])
-@register_opt2([tensor.AdvancedIncSubtensor1], 'fast_compile')
+@op_lifter([tensor.AdvancedIncSubtensor])
```

Comment:
Create a new optimization for AdvancedIncSubtensor and leave the one for AdvancedIncSubtensor1 alone.

There is no chance that this will work.